src/main/java/me/crespel/karaplan/domain/Playlist.java

```diff
@@ -58,6 +65,9 @@
 	@JoinTable(name = "playlist_song", joinColumns = { @JoinColumn(name = "FK_PLAYLIST", nullable = false) }, inverseJoinColumns = { @JoinColumn(name = "FK_SONG", nullable = false) })
 	private Set<Song> songs = Sets.newLinkedHashSet();
 
+	@ManyToMany(cascade=CascadeType.ALL, fetch = FetchType.EAGER)

```

Comment:
Beware: never use `CascadeType.ALL` on `@ManyToMany` relations, as it can delete the other side too! (in this case, users)